### DISCUSSION TITLE
配置完运行出现错误：~/java/doris/doris-run/fe/log/fe.pid (No such file or directory)

### DISCUSSION THREAD

**Discussion by chenghui-ch** [apache/doris#32731] [RFC]
大佬们，这是啥情况
<img width="895" alt="截屏2024-03-24 12 29 22" src="https://github.com/apache/doris/assets/48585272/2db713ee-f657-40a8-b4d1-972e689ccf25">

配置信息：
<img width="490" alt="截屏2024-03-23 21 56 25" src="https://github.com/apache/doris/assets/48585272/23bd4352-fe34-4418-a167-ff0e975aeba7">

**Comment by ihadoop** [comment]
你给个绝地路径

**Reply by chenghui-ch** [reply]
多谢🙏。
确实是要写绝对路径😵
